.github/workflows/deploy.yml

```diff
@@ -9,6 +9,8 @@ on:
       - develop
     types:
       - completed
+  pull_request:
+    types: [ opened, reopened, edited, synchronize ]
```

Comment:
Should we remove this one now that the CD was tested? (we don't want to deploy to staging automatically on a pull_request event as multiple pull requests  can be open and will then make the staging env unstable).